bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,22 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
-python:
-  - 2.7
+  - eggs
```

Comment:
we list the directories we want to keep using the the `cache` option; buildout runs normally downloading and decompressing all eggs to the `eggs` directory in the root of the package; we include that directory in the cache; that's all; no magic, just plain buildout with no additional configuration.

https://docs.travis-ci.com/user/caching/#How-does-the-caching-work%3F